Expo1900/Controller/AppDelegate.swift

```diff
@@ -8,9 +8,17 @@ import UIKit
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
-
-
+    
+    var isRotationControl: Bool = false
+    
+    func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask {
+        if isRotationControl == true {
+            return UIInterfaceOrientationMask.all
+        }
```

Comment:
부울 조건은 보통 이렇게 하곤합니다.
```suggestion
        if isRotationControl {
            return UIInterfaceOrientationMask.all
        }
```